pom.xml

```diff
@@ -331,12 +331,19 @@
 	                </excludes>
 	            </configuration>
 	            <executions>
-	                <execution>
-	                    <id>default-report</id>
-	                    <goals>
-	                        <goal>report</goal>
-	                    </goals>
-	                </execution>
+                    <execution>
+                        <id>default-report</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>report-aggregate</goal>
+                        </goals>
+                        <configuration>
+                            <includeCurrentProject>true</includeCurrentProject>
+                            <formats>
+                                <format>XML</format>
+                            </formats>
+                        </configuration>
+                    </execution>
```

Comment:
The `sonar.yml` job is looking for the `jacoco.xml` files in `target/site/jacoco` and `report-aggregate` puts them in `target/site/jacoco-aggregate`. You should be able to add `<outputDirectory>${project.reporting.outputDirectory}/jacoco</outputDirectory>` to override it to the expected location.

Also, just a nit - but this XML block can be indented.

Reply:
Oh yes! I missed that change in this commit. But @MikeEdgar one concern/question I have is that 
Even with these changes, in the operator `index.html` we see `api` and `common` coverage but not `operator` coverage. Same for `sync`. I think we are missing something here. What do you think?

I experimented with `report` and `report-aggregate` but had no luck.

Reply:
Something is definitely still not right... I'm checking now too.